ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -556,9 +555,9 @@ static const func_info_t func_infos[] = {
 	FN("key",                          MAY_BE_NULL | MAY_BE_LONG | MAY_BE_STRING),
 	FN("min",                          UNKNOWN_INFO),
 	FN("max",                          UNKNOWN_INFO),
-	FN("array_search",                 MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_LONG | MAY_BE_STRING),
+	FN("array_search",                 MAY_BE_FALSE | MAY_BE_LONG | MAY_BE_STRING),
```

Comment:
We should also be able to drop FN, because that's actually the default behavior (if there are refcounted types, otherwise F0).